### ISSUE TITLE
[REQUEST]Some Suggestions in Stacker

### ISSUE THREAD

**Issue by MikeAndarsons** [openbullet/OpenBullet2#763]
=>Double Click to Disable Block in Stacker.(Not Delete, Probably you can add in settings like double click to disable or delete)
=>there should be "+" Block always in Stacker in end of blocks to directly Add block So that no need to go all time upper corner to add blocks

**Comment by thisisthelongestusernameyouhaveeverseen** [comment]
it's better to be able to right click on a block and be able to disable/enable/delete/clone it or add another block after it

**Comment by MikeAndarsons** [comment]
@thisisthelongestusernameyouhaveeverseen Good idea, There should be both I mean we should be able to configure Double Click in settings to any of the action disable/enable/delete/clone, along with Right Click Options, And one more thing "Add block" option is good to be in Right Click But It also should be Constant "Add Block"== [     +    ] in Stacker.
